tests/cephfs/cephfs_utils.py

```diff
@@ -56,16 +56,29 @@ def get_clients(self, build):
                 out, rc = node.exec_command(cmd='sudo rpm -qa | grep -w attr')
                 output = out.read().decode()
                 output.split()
+                distro, rc =node.exec_command(cmd='sudo lsb_release -a')
```

Comment:
you don't need to use this command here. you can use ceph module to get the distro information like this 
https://github.com/red-hat-storage/cephci/blob/master/tests/misc_env/install_prereq.py#L44

Reply:
@yogesh-mane any update on this. lets merge this as quickly as possible. some of the inetrop suites are failing.